templates/download/core/intel-nuc.html

```diff
@@ -0,0 +1,213 @@
+{% extends "download/_base_download.html" %}
+
+{% block title %}Install Ubuntu on the Intel NUC{% endblock %}}
```

Comment:
does the Intel need a &lt;&reg;>?

Reply:
I don't think you can do superscript on the title, but I can put the registered trademark symbol in